2019-07-31-HW-XML/src/Demo.java

```diff
@@ -26,11 +30,13 @@ private static void readDataXML(Document doc) {
                 String personName = personElement.getElementsByTagName("name").item(0).getTextContent();
                 String personGender = personElement.getElementsByTagName("gender").item(0).getTextContent();
                 String personAddress = personElement.getElementsByTagName("address").item(0).getTextContent();
-
-                System.out.println("Name = " + personName);
-                System.out.println("Gender = " + personGender);
-                System.out.println("Address = " + personAddress);
+                persons.add(personName);
```

Comment:
No.
The idea was to create instances of class Person out of XML structures, but you just created strings.

So, I would expect something like 

`Person p = new Person(personName, personGender, personAddress);`